frameworks/hdfs/tools/kdc.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "/kerberos",
+  "instances": 1,
+  "cpus": 1,
+  "mem": 512,
+  "labels": {
+    "DCOS_SERVICE_NAME": "kerberos"
+  },
+  "env": {
+    "PRINCIPAL_hdfs": "hdfs@LOCAL:secret",
```

Comment:
What do these envvars do?

Reply:
Nothing of significance AFAICT.

Just committed the file as it was.

Reply:
Kk. At least gives us all a KDC to mess around with.